src/constants.js

```diff
@@ -31,4 +31,7 @@ exports.K = 20
 // Alpha is the concurrency for asynchronous requests
 exports.ALPHA = 3
 
+// Number of disjoint paths to search down
+exports.DISJOINT_PATHS = 6
```

Comment:
Shouldn't this be 16? 

According to [go-libp2p-kad-dht#146#issue-320532077](https://github.com/libp2p/go-libp2p-kad-dht/issues/146#issue-320532077)

> we actually need to query with 16 different paths

Reply:
I'm curious where the 16 came from in the original issue.

I pulled the number 6 from the S/Kademlia paper, but I agree it might be a bit low. The actual simulations from the paper don't seem very thorough to be honest, but the authors do suggest d [number of disjoint paths] = k/2: "We conclude that d = 4..8 with a k = 8..16 is a good choice for S/Kademlia". Given that we're using k = 20, maybe we should start with d = 10?

I think we'll have to do simulation and/or testing to know what the right choice actually is.

Reply:
yes! It would be nice to make some simulation tests, in order to determine the best number here. By the end of this quarter, we will hopefully have a testbed for large scale tests to the DHT. Maybe there we can have better conclusions about this.

Reply:
I will try to read the S/Kademlia paper again during the next week. I read it a long time ago, and I hope to be able to provide you better feedback once you add the remaining tests.

Reply:
New version uses d=10. Thoughts?

Reply:
I would like to have tests using several different `d`, in order to get the best option. Maybe we should discuss this in the original issue and grab opinion from other folks